packages/vite-plugin/src/plugins.ts

```diff
@@ -12,6 +12,10 @@ export interface ProjectData {
    * The file name containing the project.
    */
   fileName: string;
+  /**
+   * The path to the file containing the project
+   */
+  filePath: string;
```

Comment:
This is too confusing I think. `url` is storing the actual path and `filePath` is the url used to access the project (with no file extension etc) Maybe we could swap the two?